packages/react/package.json

```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@0no-co/graphql.web": "^1.0.4",
-    "@gadgetinc/api-client-core": "^0.15.34",
+    "@gadgetinc/api-client-core": "^0.15.35",
```

Comment:
These also have references to 0.15.34

packages/shopify-extensions/package.json
packages/react-shopify-app-bridge/package.json
packages/react-bigcommerce/package.json